### PR TITLE
Move to finn eslint config with es6 code 

### PR DESCRIPTION
**Pull request by sveisvei**
Want to use node es6 code and node 6 as target.

* used jscodeshift with js-codemod`s: ```no-vars, object-shorthand, template-literals, unchain-variables, unquote-properties, arrow-function, use-strict```
* used eslint --fix
* manual fixes for rest of errors.

### CODE REVIEW

File: shift.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+jscodeshift \
```
Comment (ivarconr) [inline shift.sh]: this file is a one-time thing and should probably not be checked in?

**Comment by ivarconr** [comment]
I guess we need to trust our tests on this one!

File: shift.sh (hunk 1 above)
Comment (sveisvei) [inline shift.sh]: Nop, just wip "snapshot". Will force all kinds of stuff later, use the force you know 🌠

File: packages/unleash-api/lib/db/event.js [hunk 2]
```diff
@@ -21,7 +22,7 @@ module.exports = function(db) {
         return db
         .select(EVENT_COLUMNS)
         .from('events')
-        .whereRaw("data ->> 'name' = ?", [name])
+        .whereRaw('data ->> \'name\' = ?', [name])
```
Comment (ivarconr) [inline packages/unleash-api/lib/db/event.js]: it might be correct linting, but not easier to read..

Comment (sveisvei) [inline packages/unleash-api/lib/db/event.js]: I can manually fix it, will have to replace \' with \", thats what you want right?

Comment (SimenB) [inline packages/unleash-api/lib/db/event.js]: We should add `avoidEscape`

Comment (SimenB) [inline packages/unleash-api/lib/db/event.js]: https://github.com/finn-no/eslint-config-finn/pull/51

**Pull request merged**